src/sentry/static/sentry/app/components/projects/missingProjectMembership.tsx

```diff
@@ -35,10 +42,11 @@ class MissingProjectMembership extends React.Component<Props, State> {
       loading: false,
       error: false,
       project,
+      team: '',
     };
   }
 
-  joinTeam(team: Team) {
```

Comment:
Instead of removing the type, can we update it?

Reply:
I updated this to use `team: string`